<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeScreen</class>
 <widget class="QWidget" name="WelcomeScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>359</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Art Of Sequence : Designer - Welcome!</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>341</width>
     <height>241</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="sizeConstraint">
     <enum>QLayout::SetMaximumSize</enum>
    </property>
    <item>
     <widget class="QLabel" name="logo_display">
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="../rc/AOSDesigner.qrc">:/images/images/welcome_logo.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="button_new_project">
        <property name="text">
         <string>New Project</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_open_project">
        <property name="text">
         <string>Open Project</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <tabstops>
  <tabstop>button_new_project</tabstop>
  <tabstop>button_open_project</tabstop>
 </tabstops>
 <resources>
  <include location="../rc/AOSDesigner.qrc"/>
 </resources>
 <connections/>
</ui>
